// 装备碎片
message EquipmentChipPB{
	required int32 equipment_chip_no = 1;  //编号
	required int32 equipment_chip_num = 2;  //数量
}

// 返回装备碎片列表
message GetEquipmentChipsResponse
{
    repeated EquipmentChipPB equipment_chips = 1;
}